### PR TITLE
Feature/io 653 sharing simple data plugin

### PR DESCRIPTION
**Pull request by nikolay-ahramovich-idf**

### CODE REVIEW

File: packages/presentation/lib/app/app_bloc.dart [hunk 1]
```diff
@@ -31,8 +34,30 @@ class _AppBloc extends BlocImpl<BaseArguments, AppData> implements AppBloc {
...
-    _popUntil(HomeScreen.page());
+  void loadPage(int pageIndex) {
+    final type = BottomNavigationItemTypeExt.toType(pageIndex);
+
+    switch (type) {
+      case BottomNavigationItemType.home:
+        _goToHomePage(pageIndex);
+        break;
+      case BottomNavigationItemType.login:
+        _goToLoginPage(pageIndex);
```
Comment (ekaterina-demeshko-idf) [inline packages/presentation/lib/app/app_bloc.dart:45]: add break

**Review by ekaterina-demeshko-idf — APPROVED** [review]

File: android/app/build.gradle [hunk 2]
```diff
@@ -68,4 +82,8 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation platform('com.google.firebase:firebase-bom:30.4.1')
+    implementation 'com.google.firebase:firebase-analytics'
+    implementation 'com.facebook.android:facebook-android-sdk:latest.release'
```
Comment (ekaterina-demeshko-idf) [inline android/app/build.gradle]: add version

Comment (nikolay-ahramovich-idf) [reply android/app/build.gradle]: fixed.

File: packages/share_movie_plugin/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```
Comment (andrei-lastochkin-idf) [inline packages/share_movie_plugin/CHANGELOG.md]: remove

File: packages/share_movie_plugin/LICENSE [hunk 4]
```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```
Comment (andrei-lastochkin-idf) [inline packages/share_movie_plugin/LICENSE]: -//-

**Review by andrei-lastochkin-idf — APPROVED** [review]

File: packages/share_movie_plugin/CHANGELOG.md (hunk 3 above)
Comment (nikolay-ahramovich-idf) [reply packages/share_movie_plugin/CHANGELOG.md]: Fixed.

File: packages/share_movie_plugin/LICENSE (hunk 4 above)
Comment (nikolay-ahramovich-idf) [reply packages/share_movie_plugin/LICENSE]: Fixed.

File: packages/share_movie_plugin/lib/const.dart [hunk 5]
```diff
@@ -0,0 +1,4 @@
+class ShareMoviePluginAndroid {
+  ShareMoviePluginAndroid._();
+  static const intentTitle = 'Movie Sharing';
```
Comment (sergei-mikhailovskii-idf) [inline packages/share_movie_plugin/lib/const.dart]: pass it as a param to the plugin. since in another app u will have other title, not Movie Sharing

**Review by sergei-mikhailovskii-idf — APPROVED** [review]

File: packages/share_movie_plugin/lib/const.dart (hunk 5 above)
Comment (nikolay-ahramovich-idf) [reply packages/share_movie_plugin/lib/const.dart]: Fixed.

**Pull request merged**